src/shared/types/augment-node-global.d.ts

```diff
@@ -3,6 +3,6 @@ declare namespace NodeJS {
 		/**
 		 * This comes from webpack.
 		 */
-		BROWSER: boolean;
+		browser: boolean;
```

Comment:
Not directly related to this PR but iirc you can use `@types/webpack-env` to have these webpack related types augmented

Reply:
Ah, this is because `process.browser` is not webpack official either, it is provided through the browser polyfill for Node.js variables.

Reply:
It seems that using `typeof window !== "undefined"` is the best alternative. It also works without polyfill or webpack itself